src/NovelBasicFillRect.cpp

```diff
@@ -6,11 +6,11 @@ namespace NovelRT::Graphics {
 
 BasicFillRect::BasicFillRect(LayeringService* layeringService,
                                        const RGBAConfig& fillColour,
-                                       const CommonArgs& args,
+                                       const Utilities::CommonArgs& args,
```

Comment:
Just commenting here as well, since we already talked about it in Discord.

`CommonArgs`, given that its part of the "public" API, should likely not be in `Utilities`